app/Repositories/H5pElasticsearchField/H5pElasticsearchFieldRepositoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Repositories\H5pElasticsearchField;
+
+use App\Models\H5pElasticsearchField;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+interface H5pElasticsearchFieldRepositoryInterface
```

Comment:
It should extend EloquentRepositoryInterface.

Reply:
Added extend for EloquentRepositoryInterface.